plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/binance/v4/BinanceSortedGroupV4.java

```diff
@@ -38,24 +37,24 @@ public class BinanceSortedGroupV4 {
     Object time;
 
     // beforeSum
-    Map<Currency, List<BinanceBeanV4>> rowsDeposit = new HashMap<>();
-    Map<Currency, List<BinanceBeanV4>> rowsWithdrawal = new HashMap<>();
-    Map<Currency, List<BinanceBeanV4>> rowsFees = new HashMap<>();
-    Map<Currency, List<BinanceBeanV4>> rowsBuySellRelated = new HashMap<>();
-    Map<Currency, List<BinanceBeanV4>> rowsRewards = new HashMap<>();
+    Map<Currency, List<BinanceBeanV4A>> rowsDeposit = new HashMap<>();
```

Comment:
naming BinanceBeanV4A and BinanceBeanV4B is not good enough. As I understand BinanceBeanV4B is for somehow different CSV file than BinanceBeanV4. So you should not use V4. You should use BinanceBeanV5 or Binance<Something>BeanV1 (new type of csv file?). Its not ideal that ve use V1 , V2, VX... anyway, so i dont thing that bringing another versioning is good idea. 

You still can use common code for these 2 beans, this is just about names. And naming it A B is not enough. Its not clear from code what is happening.